asteroid/metrics.py

```diff
@@ -342,6 +344,7 @@ def hsdi(truth, hypothesis, transformation):
         return {k: v for k, v in out if k in keep}
 
     def predict_hypothesis(self, wav):
+        wav = torch.from_numpy(wav).cuda()
```

Comment:
That does not seem right? Isn't is `.to(self.device)`?

Reply:
Oh yeah. My bad.

Reply:
fixed